README.md

```diff
@@ -410,6 +410,18 @@ section.  An appropriate title should be specified for each form.
 Nested markup required to enable the animated loading progress bar
 without requiring any external graphics.
 
+        <div id="err">
+          <div id="worried-face">&#x1F61F;</div>
```

Comment:
Is this emoticon portable to old browsers? A safer alternative would be a small image. And it could be colorful, for what it's worth.

Reply:
Here are some examples. Of course these would be styled smaller.
![icon-warning](https://user-images.githubusercontent.com/182744/44422643-9583e200-a538-11e8-9152-0e2c7b9538de.png)
![icon-check](https://user-images.githubusercontent.com/182744/44422656-a03e7700-a538-11e8-8c08-4ebb36275bc9.png)
![icon-x](https://user-images.githubusercontent.com/182744/44422670-a7658500-a538-11e8-99b9-29e1e2e45d14.png)

Reply:
The reason I went with the emoji was that I wanted to be able to show the error message without loading any additional resources in case the problem was that the resources weren't available due to a botched install or whatever. I'm pretty confident all up-to-date common browser will show this unicode codepoint correctly.  At the time I did this I wasn't really thinking about old browser support. Researching that a bit, it looks like most browsers support showing at least some emoji going back to the IE9 timeframe, which covers most of what we're concerned about. However, it isn't clear which codepoints are rendered correctly in which versions of the browser. I'm tempted to just drop the cute graphic rather than add a dependency on an external resource for this function. Any thoughts on that?